src/components/MobileNavMenu/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styles from "./styles.module.scss";
+import Link from "next/link";
+interface Props {
+    open?: boolean;
+}
+const MobileNavMenu: React.FC<Props> = ({open}) => {
+    return (
+        <nav className={styles.navigation}
+        style={open ? {right: 0} : {right: "-100%"}}>
```

Comment:
Not sure if this is do-able, and others feel free to disagree, but is there any way to make it so that the menu only comes out halfway, so the map is still partially visible? I think that might look neat. The issue with it is that the text ends up being hidden on the right, not sure how to solve that.

Reply:
I think you're right, that probably would look better. I'll fix it and post what the changes look like.